app/src/main/java/com/github/libretube/ui/views/CustomExoPlayerView.kt

```diff
@@ -474,7 +475,15 @@ abstract class CustomExoPlayerView(
         },
         BottomSheetItem(
             context.getString(R.string.sleep_timer),
-            R.drawable.ic_sleep
+            R.drawable.ic_sleep,
+            getCurrent = {
```

Comment:
```suggestion
            {
```

The explicit argument naming isn't necessary here.